### PR TITLE
Updated to Mosaic 0.6.6 release

### PR DESCRIPTION
**Pull request by d3cod3**
Thanks to [bbhtt](https://github.com/bbhtt) for fixing the update for OF 0.11.2 plus a bunch of patches

### CODE REVIEW

**Comment by hfiguiere** [comment]
```
        "module-libdaemon-source-git-url-not-http"
```

use https URL for git.

It would be `https://git.0pointer.net/clone/libdaemon.git`

File: org.d3cod3.Mosaic.appdata.xml [hunk 1]
```diff
@@ -42,14 +40,24 @@
...
+          - multi-threaded
+          - ImGui based docking GUI interface
+          - non-blocking system dialogs
+          - lua/glsl/bash live-coding
+          - pure data live-patching
+          - good selection of audio synthesis modules
+          - non-linear interactive graphical timeline
+          - warping integrated on output window object
+          - mapmap style projection mapping module
+          - [tracy profiler](https://github.com/wolfpld/tracy) for solid debug and profiling
```
Comment (hfiguiere) [inline org.d3cod3.Mosaic.appdata.xml]: URL is description usually fail validation.

also this should be list items.

Comment (d3cod3) [reply org.d3cod3.Mosaic.appdata.xml]: Thanks for the tips!

I'm on it

**Comment by hfiguiere** [comment]
urgh, self hosted git. libdaemon requires `disable-shallow-clone: true` to be set

**Comment by d3cod3** [comment]
Ok, maybe switching to archive would be better?

type: archive
url: http://0pointer.de/lennart/projects/libdaemon/libdaemon-0.14.tar.gz
sha256: fd23eb5f6f986dcc7e708307355ba3289abe03cc381fc47a80bca4a50aa6b834

**Pull request merged**